storagegroup/storagegroup.go

```diff
@@ -18,13 +18,35 @@ import (
 // StorageGroup is mutually compatible with github.com/nspcc-dev/neofs-api-go/v2/storagegroup.StorageGroup
 // message. See ReadFromMessageV2 / WriteToMessageV2 methods.
 //
-// Instances can be created using built-in var declaration.
+// Instances should be created using one of the constructors.
 //
 // Note that direct typecast is not safe and may result in loss of compatibility:
 //
 //	_ = StorageGroup(storagegroup.StorageGroup) // not recommended
 type StorageGroup storagegroup.StorageGroup
 
+// New constructs new StorageGroup from given members and total size/checksum
+// calculated for them. Member set must not be empty.
+func New(size uint64, cs checksum.Checksum, members []oid.ID) StorageGroup {
+	var res StorageGroup
+	res.SetValidationDataSize(size)
+	res.SetValidationDataHash(cs)
+	res.SetMembers(members)
+	return res
+}
+
+// Unmarshal creates new StorageGroup and makes [Version.Unmarshal].
+func Unmarshal(b []byte) (StorageGroup, error) {
```

Comment:
is it the first time we have just `Unmarshal` func (not a method)? do you plan the other structs to have it too?

Reply:
"Version"?

Reply:
#483 is for these

Reply:
i mean, will we see such funcs (not methods) for other structs that already have `Unmarshal` method?

Reply:
we can add them anytime. Im tryin to do this while traversing all packages now. Finally, yes we will